packages/widget/src/controllers/wallet-manager/manager.ts

```diff
@@ -249,6 +275,31 @@ export class WalletController implements ReactiveController {
     }
   };
 
+  private async addEvmChain(
+    chainData: ChainData,
```

Comment:
We can destruct this object to access the props directly  like: { chainId, name ...}

Reply:
done